Updates Cloud Guide With Rate Limit and CATPCHA Images (#1641)
FusionAuth/fusionauth-site: site/docs/v1/tech/installation-guide/cloud.adoc

If you have out of scope questions and have not purchased a support contract, you can find community support in the link:/community/forum/[forums] and link:/docs/v1/tech/[documentation].

== CAPTCHA and Rate Limits

FusionAuth Cloud has a number of protections in place to prevent a DoS attack and keep your services running smoothly. If the following message appears, this would indicate the end user, test, or application is making requests in excess of what is considered typical.

image::installation-guides/cloud/too-many-requests-blocked.png[Too Many Requests,width=1200,role=bottom-cropped, top-cropped]

> Prior to this message you may receive the following CAPTCHA challenges

Reviewer: ```suggestion
Prior to this message you may receive one of the following CAPTCHA challenges, or a similar one.
```

image::installation-guides/cloud/aws-captcha-preamble-text.png[AWS Preamble CAPTCHA,width=1200,role=bottom-cropped, top-cropped]
image::installation-guides/cloud/aws-captcha-challenge-puzzle.png[Puzzle Captcha AWS,width=1200,role=bottom-cropped, top-cropped]
image::installation-guides/cloud/aws-captcha-challenge-roadmap.png[Roadmap Captcha AWS,width=1200,role=bottom-cropped, top-cropped]

> To prevent either scenario please limit the number of requests you are making over a given time period or open a support ticket to whitelist your static production IP addresses as needed.

Reviewer: ```suggestion
To prevent either scenario please limit the number of requests you are making over a given time period or https://account.fusionauth.io/account/support[open a support ticket] to add your static production IP addresses to the FusionAuth Cloud allowlist as needed.
```

> 

Reviewer: ```suggestion

[NOTE.note]
====
The CAPTCHA described here is separate from the CAPTCHA configured using link:/docs/v1/tech/advanced-threat-detection/[Advanced Threat Detection] and can only be managed by the FusionAuth team.
====
```

Author: This is great!
